features/manage-position/Create.tsx

```diff
@@ -238,7 +247,16 @@ const Create = () => {
       {hash && (
         <Box py={2}>
           <Typography>
-            <strong>Tx Hash: </strong> {hash}
+            <strong>Tx Receipt: </strong>
+              {etherscanUrl ? (
```

Comment:
This displays the `etherscanUrl` or the `hash` if the `etherscanUrl` is `undefined` for any reason